### ISSUE TITLE
post-processing several SO algorithms crashes

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#757]
```python
Traceback (most recent call last):
  File "/Users/hansen/anaconda/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/Users/hansen/anaconda/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/hansen/.local/lib/python2.7/site-packages/bbob_pproc-0.7b0-py2.7.egg/bbob_pproc/__main__.py", line 298, in <module>
    rungeneric.main(args)
  File "/Users/hansen/.local/lib/python2.7/site-packages/bbob_pproc-0.7b0-py2.7.egg/bbob_pproc/rungeneric.py", line 287, in main
    rungenericmany.main(genopts + ["-o", outputdir] + args)
  File "/Users/hansen/.local/lib/python2.7/site-packages/bbob_pproc-0.7b0-py2.7.egg/bbob_pproc/rungenericmany.py", line 424, in main
    genericsettings.verbose)
  File "/Users/hansen/.local/lib/python2.7/site-packages/bbob_pproc-0.7b0-py2.7.egg/bbob_pproc/compall/ppfigs.py", line 495, in main
    ecdfs_figure_caption(target), '}']
  File "/Users/hansen/.local/lib/python2.7/site-packages/bbob_pproc-0.7b0-py2.7.egg/bbob_pproc/compall/ppfigs.py", line 86, in ecdfs_figure_caption
    r" for all functions and subgroups in #1-D. " + ( best2009text
TypeError: cannot concatenate 'str' and 'int' objects
```

**Comment by brockho** [comment]
Fixed with pull-request #758. We should probably open another issue for including again all tests because it should have been definitely caught earlier...

**Comment by dtusar** [comment]
All post processing tests are now included in the nightly build.
